### ISSUE TITLE
Training on PC

### ISSUE THREAD

**Issue by ankitpt** [leeyoshinari/YOLO_v2#25]
Hello @leeyoshinari. If I am running this code on my PC instead of a GPU, do I need to make any changes in the code? Currently upon running the code, I reach upto a point where it prints "start training" and then prints another line as
2019-03-12 13:59:39 Epoch: 6 Step: 0, train_loss: 3.3326, test_loss: 1.4714, Remain: 0:00:00

It has been stuck here for a long time. Am I doing something wrong?

**Comment by leeyoshinari** [comment]
Hi, In "train_val.py", Comment out line 45, and change line 46 to "self.sess = tf.Session()". Others, maybe you can try to change the learning rate, batch size, or restore the weights of a part of layers.

**Comment by ankitpt** [comment]
Thank you for your response. I changed the code as you instructed and also changed batch size and learning rate. Since I am only working with 3 images which are present by default in your repository I set the batch size to 1. Thereafter the training went fine. Do I also need to make any changes in test_val.py?

**Comment by barzan-hayati** [comment]
@leeyoshinari  I didn't see any comment in line 45, and line 45 was 

> config = tf.ConfigProto(gpu_options=tf.GPUOptions())

I run your program and I reach to @ankitpt result in first post.

> Hello @leeyoshinari. If I am running this code on my PC instead of a GPU, do I need to make any changes in the code? Currently upon running the code, I reach upto a point where it prints "start training" and then prints another line as
> 2019-03-12 13:59:39 Epoch: 6 Step: 0, train_loss: 3.3326, test_loss: 1.4714, Remain: 0:00:00
> 
> It has been stuck here for a long time. Am I doing something wrong?

How you(@ankitpt ) resolve this problem?
Also, running program for my system was too slow, I think it didn't run on GPU.
